plugins/wasi_threads/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+wasmedge_add_library(wasmedgePluginWasiThreads
+  SHARED
+  threadfunc.cpp
+  threadmodule.cpp
+)
+
+target_compile_options(wasmedgePluginWasiThreads
+  PUBLIC
+  -DWASMEDGE_PLUGIN
+)
+
+if(CMAKE_SYSTEM_NAME MATCHES "Darwin")
```

Comment:
Hi, Please refer to the latest master branch. We don't need to add these link options now, instead, there are some changes in the following cmakelists.txt file.